docs/release_notes/version_chart/version_chart.md

```diff
@@ -15,13 +15,14 @@ Check to make sure the Spark version and Scala version of your Prophecy Scala Li
 of your cluster!
 :::
 
-:::info
-Query this table using the following API and your ([personal access token](..%2Fmetadata%2FprophecyAPI.md)):
+You can query this table using the following API and your ([personal access token](/docs/metadata/prophecyAPI.md)):
 
 ```bash
 curl  --header 'X-Auth-Token: $PROPHECY_PAT' --location https://app.prophecy.io/api/editor/plibVersions
 ```
 
+:::note
+Prophecy versions that are labeled with `EM` are Extended Maintenance releases. For more information, see [Prophecy versions support](/docs/release_notes/version_chart/versions_support.md).
```

Comment:
i don't see any labeled with EM yet, are you waiting to add the newest release to the table for any reason?

Reply:
there are no EM versions yet. The new one will be coming out soon (within a few days / week)

Reply:
maybe it would be confusing to have the sentence/note here before it comes out